javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,79 @@
+console.log("hello");
+//building a voice assistant.
+//function called getReply(command).
+let personName = [];
+const todoList = [];
+let today = new Date();
+let date = today.getDate();
+
+
+let options = { year: 'numeric', month: 'long' };
+const formatedDate = date + ". of " + today.toLocaleString('en-US', options);
+
+let operators = [];
+
+function getReply(command) {
+    if (command == " " || typeof command !== "string") {
+        alert("Give some command Please!");
+    }
+    else if (command.includes("Hello my name is") && personName.length === 0) {
+        personName.push({ "name": command.split(" ").pop() }); //pushed object with key name.
```

Comment:
Instead of pushing, add a value to personName (since it is a string, not an array). Please also change the rest of the cases where you use personName as the array.

Reply:
Yes will change it